pages/api/residents/index.ts

```diff
@@ -19,10 +19,7 @@ const endpoint: NextApiHandler = async (
   switch (req.method) {
     case 'GET':
       try {
-        const data = await getResidents({
```

Comment:
Unclear if we have tests for the `/api/...` endpoints, this is something I'll take a look at early next week.